tests/test_sota_checkpoints.py

```diff
@@ -359,12 +430,25 @@ def skip_params(sota_data_dir):
 
 
 @pytest.fixture(autouse=True, scope="class")
-def results():
+def openvino_preinstall(openvino):
+    if openvino is not None:
+        subprocess.run("pip uninstall setuptools -y && pip install setuptools", check=True, shell=True)
```

Comment:
Why is this required?

Reply:
Without this, an AC installation error occurs. Not sure it's CI docker problem or setuptools

Reply:
Add a comment in code explaining this.